### DISCUSSION TITLE
2023/09/27/lifetime-in-rust

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#12] [General]
# 2023/09/27/lifetime-in-rust

在前面的「所有權（Ownership）」章節曾經介紹 Rust 是如何透過所有權的轉讓與出借，來決定是否要釋放不再使用的記憶體空間，但其實在介紹所有權的時候，我們有省略了一些東西。Rust 有「借（Borrow）」的設計，不需要複製整份資料，只要用 & 做個參照或切片，就能透過參照直接取用原本的資料。但是...

https://kaochenlong.com/2023/09/27/lifetime-in-rust.html

**Comment by tfing** [comment]
第三段的 "使用後釋放（use-after-free）" 應該是 "釋放後使用" ?

**Reply by kaochenlong** [reply]
對，我竟然照著字面翻譯了，我的錯
感謝提醒，已修正
